wire-library/wire-gradle-plugin/src/test/projects/custom-output/build.gradle

```diff
@@ -30,4 +30,7 @@ wire {
   custom {
     customHandler = new MyCustomHandler()
   }
+
+  java{
+  }
```

Comment:
That's a change in behavior here, no?
I don't think Java was created here

Reply:
Good catch will fix